concrete/src/Updater/Migrations/RepeatableMigrationInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations;
+
+/**
+ * Interface that migrations that can be safely re-executed should implement.
```

Comment:
So can it be dangerous, or is it safe?

Reply:
> So can it be dangerous, or is it safe?

It is safe: I added that warning before implementing (and even imagine) the `RepeatableMigrationInterface` interface.
So, I've just removed that warning.